### PR TITLE
Add implicit account examples and fix some Python types

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add implicit account examples and fix some Python types

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Running examples

### CODE REVIEW

File: bindings/python/iota_sdk/types/output_data.py [hunk 1]
```diff
@@ -21,6 +19,7 @@ class OutputData:
         output_id: With the output data corresponding output ID.
         metadata: With the output corresponding metadata.
         output: The output object itself.
+        output_id_proof: The output ID proof.
         address: The address associated with the output.
```
Comment (thibault-martinez) [inline bindings/python/iota_sdk/types/output_data.py]: remove

File: bindings/nodejs/examples/how_tos/account_output/implicit-account-creation.ts
Comment (thibault-martinez) [inline bindings/nodejs/examples/how_tos/account_output/implicit-account-creation.ts:1]: Doesn't really create the account...

Comment (Thoralf-M) [reply bindings/nodejs/examples/how_tos/account_output/implicit-account-creation.ts:1]: That's how it was called already in python :trollface: Should I just add `-address`?

Comment (Thoralf-M) [reply bindings/nodejs/examples/how_tos/account_output/implicit-account-creation.ts:1]: Renamed

**Review by thibault-martinez — APPROVED** [review]

File: bindings/python/iota_sdk/types/transaction_with_metadata.py [hunk 2]
```diff
@@ -23,12 +23,12 @@ class InclusionState(str, Enum):
         Conflicting: The transaction is conflicting.
         UnknownPruned: The transaction is unknown or already pruned.
     """
-    Pending = 'pending'
-    Accepted = 'accepted'
-    Confirmed = 'confirmed'
-    Finalized = 'finalized'
-    Conflicting = 'conflicting'
-    UnknownPruned = 'unknownPruned'
+    Pending = 'Pending'
```
Comment (DaughterOfMars) [inline bindings/python/iota_sdk/types/transaction_with_metadata.py:26]: Why?

Comment (Thoralf-M) [reply bindings/python/iota_sdk/types/transaction_with_metadata.py:26]: Serialization for this type is like this

Comment (DaughterOfMars) [reply bindings/python/iota_sdk/types/transaction_with_metadata.py:26]: But isn't it supposed to be camelCase?

Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/transaction_with_metadata.py:26]: I think this is not a TIP type so we don't really care?

Comment (DaughterOfMars) [reply bindings/python/iota_sdk/types/transaction_with_metadata.py:26]: okie dokie 🤷‍♀️

**Review by DaughterOfMars — APPROVED** [review]

**Pull request merged**